src/fundus/publishers/de/zdf.py

```diff
@@ -0,0 +1,37 @@
+import datetime
+from typing import List, Optional
+
+from lxml.cssselect import CSSSelector
+
+from fundus.parser import ArticleBody, BaseParser, ParserProxy, attribute
+from fundus.parser.utility import (
+    extract_article_body_with_selector,
+    generic_author_parsing,
+    generic_date_parsing,
+)
+
+
+class ZDFParser(ParserProxy):
+    class V1(BaseParser):
+        _div_selector = CSSSelector("div.r1nj4qn5")
+        _summary_selector = CSSSelector("p.ikh9v7p.c1bdz7f4")
```

Comment:
The articles seem to include subheadlines as well.
https://www.zdf.de/nachrichten/politik/ausland/trump-johnson-ukraine-hilfen-100.html
https://www.zdf.de/nachrichten/panorama/busunfall-klassenfahrt-jugendliche-verletzte-marburg-100.html

Would be good to add a `_subheadline_selector` as well.

Reply:
Added the `_subheadline_selector` as well. Let me know if anything is missing 👍